README.md

```diff
@@ -12,37 +20,42 @@ If you are want to create a search UI for your website then checkout our [React
 * [License](#license)
 
 ## Install
+
+### NPM/Yarn
+
+```
+npm install --save @sajari/sdk-js
 ```
-npm install --save sajari
+
+### Browser
+
+```html
+<script src="https://unpkg.com/@sajari/sdk-js@1.0.0/dist.iife/main.js"></script>
 ```
 
 ## Getting Started
 
-A quick search example using the `website` search pipeline.
+Here's a quick example that performs a search using the query "FAQ".
 
 ```javascript
-import { Client, Tracking } from "sajari";
+import { Client, DefaultSession, TrackingClick } from "@sajari/sdk-js";
 
-const client = new Client("<project>", "<collection>");
+const pipeline = new Client("", "sajari-com").pipeline("website");
+// Tracking disabled due to not handling the results.
+const session = new DefaultSession(TrackingNone, "url", {});
 
-const tracking = new Tracking();
-tracking.clickTokens("url");
-
-client.searchPipeline(
-  "website",
-  { q: "Foo Bar", resultsPerPage: "10" },
-  tracking,
-  (err, res) => {
-    console.log(err || res);
+pipeline.search({ q: "FAQ" }, session, (error, results, values) => {
+  if (error) {
```

Comment:
I probably wouldn't put this code in here. Keep the example very short. You can simply put a single comment. eg. 
```
// Handle response here
```